src/main/webapp/js/media_manager.js

```diff
@@ -961,7 +961,7 @@ export class MediaManager
 	 setVideoCameraSource(streamId, mediaConstraints, onEndedCallback, stopDesktop) 
 	 {
 		this.navigatorUserMedia(mediaConstraints, stream => {		
-			if(stopDesktop && this.secondaryAudioTrackGainNode) {
+			if(stopDesktop && this.secondaryAudioTrackGainNode && stream.getAudioTracks().length > 0) {
```

Comment:
This change just runs the related code snippet if there is audio track in the stream